roles/redhat_subscription/tasks/main.yml

```diff
@@ -21,14 +21,22 @@
 # Below we use string formatting to use a variable as the name of the file in
 # the lookup()
 # via http://stackoverflow.com/a/34239020
-#
-  - name: Get origin file
-    command: ostree admin --print-current-dir
-    register: origin_file
 
-  - name: Get refspec from origin file
-    command: grep refspec {{ origin_file.stdout }}.origin
-    register: refspec
+  - name: status of /usr/bin/ostree is known
+    shell: type -Pf /usr/bin/ostree | /bin/true
```

Comment:
This is mostly due to my own ignorance, but...is there any harm possible running the affected commands on a non-atomic host, where ostree is present?  Related, is there any benefit (testing or otherwise) to running refspec/origin related tasks?

Another thing to consider is...maybe this role is too tightly-coupled to atomic-host and should be broken down into more discrete roles.  Again, I'm totally ignorant when it comes to how these refspec/origin tasks relate to sub. man.

Alternatively, I/we could split the tasks down into (local/static) include files.  e.g.:

```YAML
- name: Atomic-specific stuff happens before sub. man. command
  include: '{{ role_dir }}/tasks/atomic_stuff_before.yml
  when: is_atomic  # should probably set this in common somewhere

- name: Register with subscription-manager
  ...

- name: Atomic-specific stuff happens after sub. man. command
  include: '{{ role_dir }}/tasks/atomic_stuff_after.yml
  when: is_atomic  # should probably set this in common somewhere
```